# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libchipcard1/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur libchipcard-0.9.1-orig/src/libchipcard/cards/ctfilesystem2.h libchipcard-0.9.1/src/libchipcard/cards/ctfilesystem2.h
--- libchipcard-0.9.1-orig/src/libchipcard/cards/ctfilesystem2.h	2008-07-26 13:48:38.000000000 +0000
+++ libchipcard-0.9.1/src/libchipcard/cards/ctfilesystem2.h	2008-07-26 14:11:32.000000000 +0000
@@ -64,6 +64,8 @@
 #include <chipcard/ctmemorycard.h>
 #include <chipcard/ctdatacache.h>
 
+#include <cstring>
+
 
 /**
  * @page cardfspage Card File System 1.0
diff -Nur libchipcard-0.9.1-orig/src/libchipcard/cards/hbcicard.h libchipcard-0.9.1/src/libchipcard/cards/hbcicard.h
--- libchipcard-0.9.1-orig/src/libchipcard/cards/hbcicard.h	2008-07-26 13:48:38.000000000 +0000
+++ libchipcard-0.9.1/src/libchipcard/cards/hbcicard.h	2008-07-26 13:52:44.000000000 +0000
@@ -39,6 +39,7 @@
 
 
 #include <string>
+#include <cstring>
 #include <chipcard/cterror.h>
 
 
